languages/php/src/BitwardenSettings.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Bitwarden\Sdk;
+
+class BitwardenSettings
+{
+    private string $api_url;
+
+    private string $identity_url;
+
+    private string $user_agent = "Bitwarden PHP SDK";
+
+    private string $device_type;
+
+    public function __construct($api_url = null, $identity_url = null, $device_type = null)
+    {
+        if (is_null($api_url))
+        {
+            $this->api_url = getenv('API_URL') ?: 'https://api.bitwarden.com';
+            $this->identity_url = getenv('IDENTITY_URL') ?: 'https://identity.bitwarden.com';
+        } else {
+            $this->api_url = $api_url;
+        }
+
+        if (is_null($identity_url))
+        {
+            $this->identity_url = getenv('IDENTITY_URL') ?: 'https://identity.bitwarden.com';
+        } else {
+            $this->identity_url = $identity_url;
+        }
```

Comment:
In the C# project we decided against falling back to env variables, I think we should keep that pattern. Especially since `API_URL` and `IDENTITY_URL` are generic.

I believe you can just have them default to null, the SDK should handle it appropriately since they are nullable.

Reply:
Yes, it works even when they are null, so I don't use any default values now.